examples/xlora_train/xlora_train_example_incomplete.ipynb

```diff
@@ -0,0 +1,515 @@
+      "            )\n",
+      "          )\n",
+      "          (norm): MistralRMSNorm((4096,), eps=1e-05)\n",
+      "        )\n",
+      "        (lm_head): Linear(in_features=4096, out_features=32000, bias=False)\n",
+      "      )\n",
+      "    )\n",
+      "    (internal_xlora_classifier): XLoraClassifier(\n",
+      "      (softmax): TemperatureScaledSoftmax(\n",
+      "        (softmax): Softmax(dim=-1)\n",
+      "      )\n",
+      "      (layers): Sequential(\n",
+      "        (0): Linear(in_features=4096, out_features=2048, bias=True)\n",
+      "        (1): ReLU()\n",
+      "        (2): Dropout(p=0.2, inplace=False)\n",
+      "        (3): Linear(in_features=2048, out_features=2048, bias=True)\n",
+      "        (4): ReLU()\n",
+      "        (5): Dropout(p=0.2, inplace=False)\n",
+      "        (6): Linear(in_features=2048, out_features=2048, bias=True)\n",
+      "        (7): ReLU()\n",
+      "        (8): Dropout(p=0.2, inplace=False)\n",
+      "        (9): Linear(in_features=2048, out_features=2048, bias=True)\n",
+      "        (10): ReLU()\n",
+      "        (11): Dropout(p=0.2, inplace=False)\n",
+      "        (12): Linear(in_features=2048, out_features=2048, bias=True)\n",
+      "        (13): ReLU()\n",
+      "        (14): Dropout(p=0.2, inplace=False)\n",
+      "        (15): Linear(in_features=2048, out_features=2048, bias=True)\n",
+      "        (16): ReLU()\n",
+      "        (17): Dropout(p=0.2, inplace=False)\n",
+      "        (18): Linear(in_features=2048, out_features=2048, bias=True)\n",
+      "        (19): ReLU()\n",
+      "        (20): Dropout(p=0.2, inplace=False)\n",
+      "        (21): Linear(in_features=2048, out_features=1440, bias=True)\n",
+      "      )\n",
+      "    )\n",
+      "  )\n",
+      ")\n"
+     ]
+    }
+   ],
+   "source": [
+    "from peft.tuners.xlora.config import XLoraConfig\n",
+    "from peft.utils.peft_types import PeftType, TaskType\n",
+    "\n",
+    "adapters = [\n",
+    "    \"lamm-mit/Zephyr_Bioinspired\",\n",
+    "    \"lamm-mit/Zephyr_CoT\",\n",
+    "    \"lamm-mit/Zephyr_Chemistry\",\n",
+    "    \"lamm-mit/Zephyr_Math\",\n",
+    "    \"lamm-mit/Zephyr_Physics\",\n",
+    "    \"lamm-mit/Zephyr_Biology\",\n",
+    "    \"lamm-mit/Zephyr_Mechanics-and-Materials\",\n",
+    "    \"lamm-mit/Zephyr_Playtupus-Logical\",\n",
+    "    \"lamm-mit/Zephyr_Protein-Mechanics\",\n",
+    "]\n",
+    "adapters = {str(i): file_name for i, file_name in enumerate(adapters)}\n",
+    "\n",
+    "peft_config = XLoraConfig(\n",
+    "    task_type=TaskType.CAUSAL_LM,\n",
+    "    peft_type=PeftType.XLORA,\n",
+    "    hidden_size=model.config.hidden_size,\n",
+    "    adapters=adapters,\n",
+    "    xlora_depth=8,\n",
+    "    xlora_size=2048,\n",
+    "    layerwise_scalings=True,\n",
+    "    xlora_dropout_p=0.2,\n",
+    ")\n",
+    "model = get_peft_model(model, peft_config).cuda()\n",
+    "print(model)\n"
```

Comment:
My personal preference is not to print the models like this: It creates a lot of output that nobody reads and distracts from the important stuff IMO.

Reply:
Sounds good! I've removed it.